### DISCUSSION TITLE
CoPilot Chat: The response was redacted... why!?

### DISCUSSION THREAD

**Discussion by Tape-Worm** [community/community#101910] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm currently asking CoPilot to write some unit tests for my constructor of a value type in C# (Visual Studio 2022).  

It generates several test cases perfectly, but when it gets to the last test case it gets about 1/2 way through and then I see this:
<img width="606" alt="image" src="https://github.com/community/community/assets/9710137/e42fb5e0-03e4-4d0b-8792-5300ab7bc610">

My code is doing absolutely NOTHING illegal, insecure, etc... There is 0 reason for this.

**Comment by jasonswearingen** [comment]
same kind of problem as OP.  this is my very first question asked to copilot chat.  very frustrating.
![image](https://github.com/community/community/assets/814134/eb4c2120-397a-4bf0-bf08-89d1429ae3e0)

**Reply by jasonswearingen** [reply]
It seems I can't discuss code with copilot when I ask about "Problem".   I restarted the conversation and avoided mentioning the word and it worked.

**Comment by ayim** [comment]
@rheapatel23's got more to share here

--
Aaron Yim
Product Manager for Github Copilot in Visual Studio
aayim@microsoft.com | github: ayim
